core/utils.ts

```diff
@@ -220,6 +232,18 @@ export function validate(compiledGraph: protos.ICompiledGraph): protos.IGraphErr
         });
       });
     }
+
+    // ignored properties in tables
+    if (!!ignoredProps[node.type]) {
+      ignoredProps[node.type].forEach(key => {
+        if (isPropertyExist(node[key])) {
```

Comment:
is this `node.hasOwnProperty(key)`?

Reply:
not quite clear what you mean?! in this function we check for empty arrays and empty objects...